frontend/api/class-interfaces/recipes/recipe.ts

```diff
@@ -77,8 +77,8 @@ export class RecipeAPI extends BaseCRUDAPI<Recipe, CreateRecipe> {
     return await this.requests.post<Recipe | null>(routes.recipesTestScrapeUrl, { url });
   }
 
-  async createOneByUrl(url: string) {
-    return await this.requests.post<string>(routes.recipesCreateUrl, { url });
+  async createOneByUrl(url: string, include_tags: boolean) {
+    return await this.requests.post<string>(routes.recipesCreateUrl, { url, include_tags });
```

Comment:
Should be camelCased

Reply:
If I change this, the backend is unable to translate the camel case into pascal case. It works fine for other routes but I'm missing it on this one.

Reply:
In the schema model you need to derive from the base `MealieModel` instead of `BaseModel` that will take care of the casing.